test/common.config

```diff
@@ -1,6 +1,8 @@
 // This file requires that params.liftover_direction and params.variant_caller be defined
 
 params {
+    output_dir = "/hot/project/method/AlgorithmEvaluation/BNCH-000142-GRCh37v38/test/output"
```

Comment:
**question (blocking):** Is this right? I thought we left this blank to either force the user to supply it at runtime or let NFTest fill it in.

Reply:
I was struggling to figure out where NFTest determines output directories so I set `output_dir` along with `NFT_OUTPUT` and `NFT_TEMP` but I can remove this line.

Reply:
@yashpatel6 Can you weigh in on the "correct" thing to do here?

Reply:
I think we should generally let NFTest handle figuring out where the output directory should be since it's set up to be able to figure out the directory by itself. Hard-coding it here could also cause issues if other users try to work on and test the pipeline due to permission issues and outputs not showing up in the development directories